website_portal/README.rst

```diff
@@ -0,0 +1,32 @@
+==============
+Website Portal
+==============
+
+Backport (with adjustments) of the ``website_portal`` module of Odoo 9.
+
+This module allows your customers to manage their account from the website
+interface, without the need to access the client backend.
+
+Credits
```

Comment:
Add "Usage" section with try on runbot button (and maybe add some content here):
```
Usage
=====

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/186/8.0
```